torch/distributed/nn/functional.py

```diff
@@ -316,7 +308,7 @@ def forward(ctx, group, out_tensor_list, *tensors):
         else:
             dist.all_to_all(
                 out_tensor_list,
-                list(tensor.contiguous() for tensor in tensors),
+                list(tensors),
```

Comment:
We don't need `.contiguous` here?

Reply:
No, this is forward and we don't need it. As long as we add contiguous in the backward, we are good to go.